en/docs/apis/swaggerfiles/organization-mgt-api-v1.yaml

```diff
@@ -2,14 +2,14 @@
 openapi: 3.0.0
 info:
   title: WSO2 Identity Server - Organization Management API Definition
```

Comment:
Will this be the tile of the doc?
It says `WSO2 Identity Server`

Reply:
Updated to `WSO2 Private CIAM Cloud`